api/authorization-api-1_0.md

```diff
@@ -260,22 +365,23 @@ A Reason Field is a JSON object that has keys and values of type `string`. The f
   "en": "location restriction violation"
 }
 ~~~
+{: #reason-example title="Example Reason"}
 
 ###### Reason Object {#reason-object}
 A Reason Object specifies a particular reason. It is a JSON object that has the following fields:
 
-id:
-: REQUIRED. A numeric value of that specifies the reason within the scope of a particular response
+`id`:
+: REQUIRED. A numeric value of that specifies the reason within the scope of a particular response.
```

Comment:
Why make it a numeric value?

Reply:
Leftover from Atul's spec. I am happy with this being a string actually.

Reply:
discussed on the call, and changed to string